src/utils/beneficial.owner.gov.ts

```diff
@@ -0,0 +1,137 @@
+import { NextFunction, Request, Response } from "express";
+import { Session } from "@companieshouse/node-session-handler";
+import { logger } from "../utils/logger";
+import { saveAndContinue } from "../utils/save.and.continue";
+import { ApplicationDataType } from "../model";
+import { getFromApplicationData, mapDataObjectToFields, mapFieldsToDataObject, prepareData, removeFromApplicationData, setApplicationData } from "../utils/application.data";
+import {
+  AddressKeys,
+  BeneficialOwnerNoc,
+  HasSamePrincipalAddressKey,
+  ID,
+  InputDateKeys,
+  IsOnSanctionsListKey,
+  NonLegalFirmNoc
+} from "../model/data.types.model";
+import { PrincipalAddressKey, PrincipalAddressKeys, ServiceAddressKey, ServiceAddressKeys } from "../model/address.model";
+import { StartDateKey, StartDateKeys } from "../model/date.model";
+import { BeneficialOwnerGovKey, BeneficialOwnerGovKeys } from "../model/beneficial.owner.gov.model";
+import { v4 as uuidv4 } from "uuid";
+
+export const getBeneficialOwnerGov = (req: Request, res: Response, next: NextFunction, currentPage: string, backLinkPage: string) => {
```

Comment:
next not used in page. Could add try / catch.

Reply:
will remove next, @AndreKramer7654 try catch not added as an error is not thrown on render method , only on methods that get data from session ex `getFromApplicationData`